test/types.go

```diff
@@ -28,7 +28,19 @@ import (
 const StorageClassName = "e2e-storage-class"
 const StorageClassName2 = "e2e-storage-class-2"
 const CSIStorageClassName = "e2e-csi-storage-class"
-const FeatureCSI = "EnableCSI"
+const FEATURE_CSI = "EnableCSI"
+const VANILLA_ZFS = "vanilla-zfs"
```

Comment:
Should we keep consistent naming conventions with previous variables?
FEATURE_CSI  -> FeatureCSI
VANILLA_ZFS  -> VanlillaZFS
AZURE_CSI ->  AzureCSI
AWS_CSI-> AWSCSI

Reply:
It's fixed, thanks.